app/models/user.rb

```diff
@@ -1,5 +1,7 @@
 class User < ApplicationRecord
-  attr_accessor :remember_token, :activation_token
+  has_many :microposts, dependent: :destroy
+  default_scope ->{order(created_at: :desc)}
```

Comment:
Trả lời câu hỏi: mục đích sử dụng default_scope là gì?
```suggestion
  default_scope ->{order created_at: :desc}
```

Reply:
defaut_scope : taọ ra các class method truy xuất dữ liệu